src/errors/test-run/templates.js

```diff
@@ -102,6 +114,20 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.uploadElementIsNotFileInputError]: err => markup(err, `
+        Upload action argument does not contain a file input element.
```

Comment:
Am I right that the action argument is a selector here?
If so, it *matches* rather than *contains* 
```
        The specified selector does not match a file input element.
```
Or we probably want to be more specific about the `upload` action?